bridgeservice/bridge.go

```diff
@@ -968,7 +968,12 @@ func (b *BridgeService) getFirstL1InfoTreeIndexForL1Bridge(ctx context.Context,
 
 	root, err := b.bridgeL1.GetRootByLER(ctx, lastInfo.MainnetExitRoot)
 	if err != nil {
-		return 0, err
+		b.logger.Infof("failed to get root by LER for L1: %v, lastInfo MainnetExitRoot: %v", err, lastInfo.MainnetExitRoot)
+		root, err = b.bridgeL1.GetLastRoot(ctx)
+		if err != nil {
+			b.logger.Errorf("failed to get last root for L1: %v", err)
```

Comment:
It should be logged at the top lvl, so maybe we can remove this log from here, but just provide enough context in the err msg itself. WDYT?